description.md

```diff
@@ -20,6 +19,43 @@ Each topic in this file contains a small description and links to the files cove
 
 ### Drills
 
+Drills are educational activities designed to reinforce learning through repetitive practice.
+The primary goal of drills is to promote mastery and internalisation of skills or knowledge.
+By repeating tasks or exercises, students can reinforce their understanding and enhance their ability to apply learned concepts effectively.
+However, it's important to balance drills with other types of learning activities to maintain student engagement and prevent monotony.
+For example, drills can be combined with homework assignments, group discussions, and other activities to provide a more varied learning experience.
+The difference is that while drills are repetitive, simple, and focused on a single skill or concept, homework activities can be more complex and open-ended.
+
+The effectiveness of drills in education depends on various factors, including the type of drill, the subject matter, and the student's learning style.
```

Comment:
What is "subject matter"?

Reply:
For example, computer science, literature, maths, etc.
But I will delete this line, it's doesn't bring anything new.